google-daemon/usr/share/google/google_daemon/accounts_manager.py

```diff
@@ -80,9 +79,7 @@ def Main(self):
           time.sleep(5)
 
         # Write the etag to pass to parent
-        json_tags = json.dumps(
-            [self.desired_accounts.attributes_etag,
-             self.desired_accounts.instance_sshkeys_etag])
+        json_tags = json.dumps([self.desired_accounts.etag])
```

Comment:
It doesn't look to me like it is ever saved to a file -- it is sent over a pipe from the forked child to the parent, so the two ends of the pipe should always be running the same binary.

I guess it's fine either way. LGTM.